tests/integration/go_ethereum/test_goethereum_http.py

```diff
@@ -1,5 +1,8 @@
 import pytest
 
+from tests.utils import (
+    get_open_port,
```

Comment:
Realizing this is a good candidate for an `eth-utils` function.  We seem to re-implement and use it in multiple code bases.

Alternatively, we could start using the `unused_tcp_port_factory` and `unused_tcp_port` fixtures provided by the `pytest-asyncio` library: https://pypi.org/project/pytest-asyncio/

Reply:
This should perhaps be done in another PR.